pom.xml

```diff
@@ -612,20 +713,19 @@
       </plugin>
       <plugin>
         <groupId>org.fusesource.hawtjni</groupId>
-        <artifactId>maven-hawtjni-plugin</artifactId>
-        <version>1.14</version>
+        <artifactId>hawtjni-maven-plugin</artifactId>
+        <version>1.18</version>
         <executions>
           <execution>
             <id>build-native-lib</id>
             <configuration>
               <name>${jniLibName}</name>
               <nativeSourceDirectory>${generatedSourcesDir}</nativeSourceDirectory>
-              <libDirectory>${project.build.outputDirectory}</libDirectory>
-              <!-- We use Maven's artifact classifier instead.
-                   This hack will make the hawtjni plugin to put the native library
-                   under 'META-INF/native' rather than 'META-INF/native/${platform}'. -->
-              <platform>.</platform>
-              <verbose>true</verbose>
+              <customPackageDirectory>${templateDir}</customPackageDirectory>
+              <windowsBuildTool>msbuild</windowsBuildTool>
+              <windowsCustomProps>true</windowsCustomProps>
+              <windowsPlatformToolset>v140</windowsPlatformToolset>
```

Comment:
Can we use v142 (the latest one)?

Reply:
Can you explain why v140 is not good enough ? I am not very into development on windows ;)

Reply:
I always prefer the latest VS libraries to avoid conflicts in various Windows 10 releases. So v142 is the latest one right now. Just a suggestion. :D

Reply:
will this also work on windows server 2008 for example ?

Reply:
If correct libraries are installed then it'll.

I can test compatibility on old systems if you want,

Reply:
that would be amazing.

Reply:
Windows Server 2008 R2 EOF January 2020. I will test on Windows Server 2012 R2 with the latest snapshot available.

Reply:
Sorry for the late reply. It works fine on Windows Server 2012 R2. Cheers! :D

Reply:
if you are interested please submit a pr.